create_tailscale_sysext.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+export ARCH="${ARCH-x86-64}"
+SCRIPTFOLDER="$(dirname "$(readlink -f "$0")")"
+
+if [ $# -lt 2 ] || [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "Usage: $0 VERSION SYSEXTNAME"
+  echo "The script will download tailscale binaries and create a sysext squashfs image with the name SYSEXTNAME.raw in the current folder."
+  echo "A temporary directory named SYSEXTNAME in the current folder will be created and deleted again."
+  echo "All files in the sysext image will be owned by root."
+  echo "To use arm64 pass 'ARCH=arm64' as environment variable (current value is '${ARCH}')."
+  "${SCRIPTFOLDER}"/bake.sh --help
+  exit 1
+fi
+
+VERSION="$1"
+SYSEXTNAME="$2"
+
+if [ "${ARCH}" = "x86-64" ]; then
+  ARCH="amd64"
+fi
+
+tarball="$(curl -fsSL "https://pkgs.tailscale.com/stable/?v=${VERSION}&mode=json" | jq -r ".Tarballs.${ARCH}")"
+
+URL="https://pkgs.tailscale.com/stable/${tarball}"
+
+TMP_DIR="$(mktemp -d)"
+
+#trap 'rm -rf "${TMP_DIR}"' EXIT
+
+curl -o "${TMP_DIR}/tailscale-amd64.tar.gz" -fsSL "${URL}"
+
+mkdir -p "${TMP_DIR}/extracted"
+tar xf "${TMP_DIR}/tailscale-amd64.tar.gz" -C "${TMP_DIR}/extracted" --strip-components=1
+
+rm -rf "${SYSEXTNAME}"
+
+mkdir -p "${SYSEXTNAME}"/usr/local/{bin,sbin,lib/{systemd/system,extension-release.d}}
+
+mv "${TMP_DIR}/extracted/tailscale" "${SYSEXTNAME}/usr/local/bin/tailscale"
+mv "${TMP_DIR}/extracted/tailscaled" "${SYSEXTNAME}/usr/local/sbin/tailscaled"
+mv "${TMP_DIR}/extracted/systemd/tailscaled.service" "${SYSEXTNAME}/usr/local/lib/systemd/system/tailscaled.service"
+
+sed -i 's/--port.*//g' "${SYSEXTNAME}/usr/local/lib/systemd/system/tailscaled.service"
+
```

Comment:
Do you want to make the service start on load or only when the user explicitly has configured it? Normally it's ok to make it start on load when it errors out on missing configuration.

Reply:
I don't really know what is the best way :/
I don't have enough experience with sysext in production to know what will be better.

Reply:
In the README we have an example on how to enable the service if the sysext image doesn't pre-enable it. We can leave it disabled if unsure.

Reply:
This PR will be a first shoot.
Let's see what will be the usage.